qutebrowser/browser/downloads.py

```diff
@@ -1238,3 +1238,11 @@ def rowCount(self, parent=QModelIndex()):
             # We don't have children
             return 0
         return len(self.downloads)
+
+    def running_downloads(self):
+        """Return the amount of still running downloads.
+
+        Return:
+            The number of unfinished downloads.
+        """
+        return len([1 for download in self.downloads if not download.done])
```

Comment:
The usual way this kind of thing is done is `sum(1 for download in self.downloads if not download.done)` - it will get you the same, but without building a list.